demo/vl-rich-data.html

```diff
@@ -317,6 +318,41 @@ <h2>Project details</h2>
           });
         </script>
       </vl-demo>
+
+      <vl-demo data-vl-title="Rich data zonder resultaten">
+        <vl-rich-data id="rich-data-no-content" data-vl-filter-closable>
+          <span slot="no-content">Geen resultaten</span>
+          <div is="vl-search-filter" data-vl-alt slot="filter">
+            <form id="rich-data-no-content-table-filter-form" novalidate>
+              <section>
+                <h2>Doorzoek projecten</h2>
+                <div>
+                  <p is="vl-form-annotation" for="filterOpId">Project id</p>
+                  <input is="vl-input-field" id="filterOpIdNoContent" type="text" name="id" value="" data-vl-block />
+                </div>
+              </section>
+              <section>
+                <h2>Project details</h2>
+                <div>
+                  <p is="vl-form-annotation" for="filterOpNaamProject">Project naam</p>
+                  <input is="vl-input-field" id="filterOpNaamProjectNoContent" type="text" name="name" value="" data-vl-block />
+                </div>
+                <div>
+                  <p is="vl-form-annotation" for="filterOpNaamManager">Manager familienaam</p>
+                  <input is="vl-input-field" id="filterOpNaamManagerNoContent" type="text" name="manager.lastName" value="" data-vl-block />
+                </div>
+              </section>
+              <div>
+                <button is="vl-button" type="button">Zoeken</button>
+              </div>
+            </form>
+            <div>
+              <button is="vl-button-link" type="reset" form="rich-data-table-filter-form">Zoekopdracht verwijderen</button>
+            </div>
+          </div>
+          <vl-pager id="rich-data-no-content-table-filter-sorting-paging-pager" slot="pager" data-vl-total-items="0" data-vl-items-per-page="10" data-vl-current-page="1" adata-vl-lign-center></vl-pager>
+        </vl-rich-data>
+      </vl-demo>
```

Comment:
Demo is overbodig want wordt niet gebruikt in de e2e testen, voldoende dekking via unit test dat initieel het no-content slot element getoond wordt.